photoprism/.env-dist

```diff
@@ -0,0 +1,108 @@
+PHOTOPRISM_TRAEFIK_HOST=photoprism.example.com
+PHOTOPRISM_VERSION=230719
+PHOTOPRISM_INSTANCE=
+TIMEZONE=Etc/UTC
+
+## The following can be named docker volumes OR host mounted absolute paths.
+## Make sure to create these directories before install.
+##
+## Where to store your original media files:
+PHOTOPRISM_ORIGINALS_VOLUME=/tmp/images
+##
+## Optionally, you can add additional media volumes, but be sure to also add
+## them to the `photoprism` service in `docker-compose.yaml` as well.
+## Example:
+# PHOTOPRISM_HOST_MEDIA_VOLUME_2=/tmp/more-images
+# PHOTOPRISM_CONTAINER_MEDIA_DIR_2=more-images
+##
+## Optionally, you can add a base folder from which files can be imported
+## to originals, but be sure to uncomment the corresponding line in the 
+## `photoprism` service in `docker-compose.yaml` as well.
+# PHOTOPRISM_HOST_IMPORT_VOLUME=/tmp/import
+
+## Traefik htpasswd encoded authentication
+## (automatically set via `make config`):
+PHOTOPRISM_HTTP_AUTH=
+## Traefik IP whitelist filter to block access to the client interface:
+## (does not affect peer connections)
+PHOTOPRISM_IP_SOURCERANGE=0.0.0.0/0
+
+
+PHOTOPRISM_ADMIN_USER=admin
+# initial admin password (8-72 characters)
+PHOTOPRISM_ADMIN_PASSWORD=password
+# server URL in the format "http(s)://domain.name(:port)/(path)"
+PHOTOPRISM_SITE_URL=http://localhost:2342/
```

Comment:
this default URL seems wrong

Reply:
it seems this might be cosmetic, but I corrected it anyway